ChiTech/ChiPhysics/FieldFunction/export_multiple_vtk.cc

```diff
@@ -75,7 +73,7 @@ void chi_physics::FieldFunction::
 
   //============================================= Populate VTK points
   auto points = vtkSmartPointer<vtkPoints>::New();
-  int vc=0;
+  int64_t vc=0;
```

Comment:
vc ? vertex count ? maybe more explicit var naming?

Reply:
Renamed to `vertex_count`.